src/server/models.py

```diff
@@ -125,18 +125,39 @@ class SalesForceDonations(Base):
 
 
 class Volgistics_Shifts(Base):
```

Comment:
@urirot do we still use this table?

Reply:
Uri confirmed that we don't, so removed it and committed.